orthanc/orthanc-raw/config/dicom.json

```diff
@@ -2,10 +2,26 @@
   "DicomAet" : "${ORTHANC_RAW_AE_TITLE}",
 
   "DicomModalities" : {
-    "UCVNAQR" : {
-      "AET" : "${VNAQR_AE_TITLE}",
-      "Port" : "${VNAQR_DICOM_PORT}",
-      "Host" : "${VNAQR_IP_ADDR}",
+    "UCPRIMARYQR" : {
+      "AET" : "${PRIMARY_DICOM_AE_TITLE}",
+      "Port" : "${PRIMARY_DICOM_PORT}",
+      "Host" : "${PRIMARY_DICOM_IP_ADDR}",
+      "Manufacturer" : "Generic",
+      "AllowEcho" : true,
+      "AllowFind" : false,
+      "AllowFindWorklist" : false,
+      "AllowGet"  : false,
+      "AllowMove" : false,
+      "AllowStore" : true,
+      "AllowStorageCommitment" : false,
+      "AllowTranscoding" : true,
+      "UseDicomTls" : false,
+      "Timeout" : 60
+    },
+    "UCSECONDARYQR" : {
+      "AET" : "${SECONDARY_DICOM_AE_TITLE}",
+      "Port" : "${SECONDARY_DICOM_PORT}",
+      "Host" : "${SECONDARY_DICOM_IP_ADDR}",
```

Comment:
what happens if we don't define the secondary QR in a deployment, I'm hoping that we just can't use that QR but it orthanc raw still runs?

If not, we could have the default values to be the same as the primary, and check to see if the primary and secondary QR are the same, and then do the early exit there?

Reply:
oh good point - I just tried it with the system tests and orthanc raw failed to start. Yeah defaulting to same as primary it a good idea

Reply:
the `SECONDARY_DICOM_SOURCE_` vars now [default](https://github.com/SAFEHR-data/PIXL/blob/6014fcd4f15957cf3e86c62acb49b6b1279ab406/docker-compose.yml#L200-L202) to `PRIMARY_DECOM_SOURCE_` values: